crates/shared/src/price_estimation/competition.rs

```diff
@@ -114,13 +131,13 @@ impl<T: Send + Sync + 'static> RacingCompetitionEstimator<T> {
 
             let best_index = results
                 .iter()
-                .map(|(_, _, result)| result)
+                .map(|(_, result)| result)
                 .enumerate()
                 .max_by(|a, b| compare_results(a.1, b.1))
                 .map(|(index, _)| index)
                 .unwrap();
-            let (stage_index, estimator_index, result) = &results[best_index];
-            let (estimator, _) = &self.inner[*stage_index][*estimator_index];
+            let (estimator_index, result) = &results[best_index];
```

Comment:
Not sure if you are aware, but you can further destructure this in order to keep the separate indices:

```suggestion
            let ((stage_index, estimator_index), result) = &results[best_index];
```

Reply:
I wanted to avoid overloading the `estimator_index` variable with both the tuple and just the index within a stage, so I preferred not further destructuring it.

Reply:
Was more of a "just so you know". I think its perfectly fine as-is.